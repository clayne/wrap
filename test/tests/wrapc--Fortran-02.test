wrapc | /dev/null | | hello_02.f90 | 0
