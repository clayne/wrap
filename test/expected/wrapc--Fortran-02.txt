!
! Fortran (formerly FORTRAN, derived from "Formula Translation") is a
! general-purpose, imperative programming language that is especially suited to
! numeric computation and scientific computing.
!
! Originally developed by IBM in the 1950s for scientific and engineering
! applications, Fortran came to dominate this area of programming early on and
! has been in continuous use for over half a century in computationally
! intensive areas such as numerical weather prediction, finite element
! analysis, computational fluid dynamics, computational physics and
! computational chemistry.
!
! It is a popular language for high-performance computing and is used for
! programs that benchmark and rank the world's fastest supercomputers.
program Hello
print *, "hello, world"
end program Hello
